### PR TITLE
dev-db/firebird: Various syntax improvements

### PR DESCRIPTION
**Pull request by wltjr**
A variety of syntax improvements per PR #101. Still can improve use flag
conditionals and a few other things mentioned in the PR not included.

### CODE REVIEW

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 EAPI=5
 
-inherit flag-o-matic eutils autotools multilib user readme.gentoo versionator
+inherit autotools eutils flag-o-matic  multilib readme.gentoo user versionator
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: Accidental extra space before multilib

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 2]
```diff
@@ -13,13 +13,13 @@ DESCRIPTION="A relational database offering many ANSI SQL:2003 and some SQL:2008
...
 
 LICENSE="IDPL Interbase-1.0"
 SLOT="0"
-KEYWORDS="~amd64 -ia64 ~x86"
+KEYWORDS="~amd64 ~x86"
 
-IUSE="doc client superserver xinetd examples debug"
+IUSE="debug doc client examples superserver xinetd"
 REQUIRED_USE="
 	client? ( !superserver )
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: I think this is equal to `?? ( client superserver xinetd )`

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 3]
```diff
@@ -48,7 +48,7 @@ pkg_setup() {
 	enewuser firebird 450 /bin/bash /usr/$(get_libdir)/firebird firebird
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: This already turned out in postgresql. You shouldn't use /bin/bash here because some minimal systems don't have /bin/bash installed.

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 4]
```diff
@@ -58,7 +58,7 @@ src_unpack() {
 	unpack "${MY_P}.tar.bz2"
 	if use doc; then
 		# Unpack docs
-		mkdir "${WORKDIR}/manuals" && cd "${WORKDIR}/manuals" || die
+		mkdir "manuals" && cd "manuals" || die
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: Double && || logic is hard to follow. Better do two commands, and two || dies.

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 5]
```diff
@@ -126,7 +126,7 @@ src_compile() {
 }
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: You shouldn't alter MAKEOPTS, just pass extra arguments to `emake`. I don't think the subst works with plain `-j` (no number). Use multiprocessing.eclass to parse MAKEOPTS.

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 6]
```diff
@@ -238,17 +238,16 @@ pkg_config() {
...
 		# if we have scurity2.fdb already, back it 1st
-		if [ -f "${ROOT}/etc/firebird/security2.fdb" ] ; then
-			cp "${ROOT}/etc/firebird/security2.fdb" "${ROOT}/etc/firebird/security2.fdb.old"
-		fi
-		gbak -B "${ROOT}/etc/firebird/security.gdb" "${ROOT}/etc/firebird/security.gbk"
-		gbak -R "${ROOT}/etc/firebird/security.gbk" "${ROOT}/etc/firebird/security2.fdb"
-		mv "${ROOT}/etc/firebird/security.gdb" "${ROOT}/etc/firebird/security.gdb.old"
-		rm "${ROOT}/etc/firebird/security.gbk"
+		[[ -f "${ROOT}/etc/firebird/security2.fdb" ]] &&
+			cp "${ROOT}/etc/firebird/security2.fdb" "${ROOT}/etc/firebird/security2.fdb.old" || die
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: This is going to die if "${ROOT}/etc/firebird/security2.fdb" doesn't exist. That's why you should use `if ...; then ... || die; fi`

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 7]
```diff
@@ -259,26 +258,26 @@ pkg_config() {
...
-		echo "localhost.localdomain" >> "${ROOT}/etc/hosts.equiv"
+	if [ grep -q 'localhost.localdomain$' "${ROOT}/etc/hosts.equiv" ] ; then
+		echo "localhost.localdomain" >> "${ROOT}/etc/hosts.equiv" || die
 		einfo "Added localhost.localdomain to ${ROOT}/etc/hosts.equiv"
 	fi
 
 	# add 'localhost' to the hosts.equiv file...
-	if [ grep -q 'localhost$' "${ROOT}/etc/hosts.equiv" 2>/dev/null ] ; then
-		echo "localhost" >> "${ROOT}/etc/hosts.equiv"
+	if [ grep -q 'localhost$' "${ROOT}/etc/hosts.equiv" ] ; then
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: This is still invalid with the `[ ]` around it.

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild [hunk 8]
```diff
@@ -259,26 +258,26 @@ pkg_config() {
...
+		echo "localhost" >> "${ROOT}/etc/hosts.equiv" || die
 		einfo "Added localhost to ${ROOT}/etc/hosts.equiv"
 	fi
 
 	HS_NAME=`hostname`
-	if [ grep -q ${HS_NAME} "${ROOT}/etc/hosts.equiv" 2>/dev/null ] ; then
-		echo "${HS_NAME}" >> "${ROOT}/etc/hosts.equiv"
+	if [ grep -q ${HS_NAME} "${ROOT}/etc/hosts.equiv" ] ; then
```
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: Likewise.

**Comment by wltjr** [comment]
You are welcome to make the rest of the changes being a gentoo developer. I am not interested in continue revisions of ebuilds that do not close any bugs nor make any progress. No further comments from me on this.

File: dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild (hunk 2 above)
Comment (mgorny) [inline dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild]: Note to self: `^^ ( ... )` actually.

**Comment by mgorny** [comment]
```
commit 9b00135f4696e539a3cbee711ac687f4f9ded105
Author: William L. Thomson Jr <wlt@o-sinc.com>
Date:   Sun Oct 4 17:44:18 2015 -0400

    dev-db/firebird: Various syntax improvements and fixes
    
    With second batch of fixes by Michał Górny.
    
    Pull-Request: https://github.com/gentoo/gentoo/pull/154
